app/src/components/market/common/add_compound_service/index.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { getCTokenForToken } from '../../../../util/tools'
+import { IconTick } from '../../../common/icons'
+import { CompoundIcon } from '../../../common/icons/currencies/CompoundIcon'
+
+const Wrapper = styled.div`
+  border-radius: 4px;
+  border: ${({ theme }) => theme.borders.borderLineDisabled};
+  padding: 18px 25px;
+  margin-bottom: 20px;
+`
+
+const Title = styled.h2`
+  color: ${props => props.theme.colors.textColorDark};
+  font-size: 16px;
+  letter-spacing: 0.4px;
+  line-height: 1.2;
+  margin: 0 0 20px;
+  font-weight: 400;
+`
+
+const DescriptionWrapper = styled.div`
+  align-items: center;
+  display: flex;
+`
+
+const CheckService = styled.div<{ isServiceChecked: boolean }>`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  text-align: center;
+  border: 1px solid ${props => (props.isServiceChecked ? props.theme.colors.transparent : props.theme.colors.tertiary)};
+  background-color: ${props =>
+    props.isServiceChecked ? props.theme.colors.clickable : props.theme.colors.mainBodyBackground};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  &:hover {
+    border: 1px solid ${props => (props.isServiceChecked ? 'none' : props.theme.colors.tertiaryDark)};
+    cursor: pointer;
+  }
+  &:active {
+    border: none;
+  }
+  path {
+    fill: ${props =>
+      props.isServiceChecked ? props.theme.colors.mainBodyBackground : props.theme.textfield.textColorDark};
+  }
+`
+
+const ServiceWrapper = styled.div`
+  color: ${props => props.theme.colors.textColorLightish};
+  font-size: ${props => props.theme.textfield.fontSize};
+  letter-spacing: 0.2px;
+  line-height: 1.4;
+  display: flex;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-pack: justify;
+`
+
+const ServiceIconWrapper = styled.div`
+  display: flex;
+  padding-right: 16px;
+  text-align: center;
+  -webkit-box-align: center;
+`
+
+const ServiceTextWrapper = styled.div`
+  width: 90%;
+`
+
+const ServiceCheckWrapper = styled.div`
+  width: 10%;
+  color: transparent;
```

Comment:
I dont think defining static values to width here is neccesary. I would suggest just adding 
justify-content: space-between; to the parent element that is 100% and omitting width for these two values. The reason for this is because 10% is slightly more than 40px and creates this unnecessary margin...
![Screenshot 2021-01-20 at 23 28 34](https://user-images.githubusercontent.com/33226956/105248740-3ac29a80-5b77-11eb-9d28-03368aca25f0.png)

Reply:
Fixed